app.rb

```diff
@@ -49,36 +50,28 @@ def perform(country_slug, legislature_slug, source = nil)
     branch_parts = [country_slug, legislature_slug, Time.now.to_i]
     branch = branch_parts.join('-').parameterize
     message = "#{country.name}: Refresh from upstream changes"
-    output = ''
-    child_status = nil
+    command = nil
```

Comment:
This raises a few warning signs to me. In general, assigning a placeholder that you're going to re-assign later is quite suspicious, and here's not really obvious why it's needed. 

I think it's also a little badly named, as it's named for what it _is_ (which is obvious as it's going be an ExternalCommand object), rather than what it's doing (i.e. it's like naming your variables `integer` or `string`), and leads to some confusion when it's later used again at line 68, even though there's been several other commands executed in the meantime. 

Perhaps it would be clearer and simpler to hoist the declaration of this command (if not others) out of within the `with_git_repo` blocks, and then only trigger the `.run` inside?